iFixit/react-commerce
# Upgrade Strapi to v4.6.2

closes #1364 

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley @masonmcelvain 

## QA

1. Login into the [strapi instance for this branch](https://fix-draft-content-type-visibility.govinor.com/admin)
2. Go to settings and check that the Strapi version is `v4.6.2`
3. Verify that bug described by #1364 is fixed, meaning that draft product lists are not rendered among children of their parents
